Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -872,31 +873,43 @@ def get_entries_for_fetched_incident(ticket_id, should_get_comments, should_get_
 def fetch_incidents(query, id_offset, should_get_attachments, should_get_comments, should_mirror_in, should_mirror_out,
                     comment_tag, attachment_tag, fetch_by_created=None):
     last_run = demisto.getLastRun()
-    demisto.debug(f"last_run: {last_run}" if last_run else 'last_run is empty')
-    if last_run and last_run.get("idOffset"):
-        id_offset = last_run.get("idOffset")
+    demisto.debug(f'last_run: {last_run}' if last_run else 'last_run is empty')
+    last_created_time = ''
+    if last_run:
+        id_offset = last_run.get('idOffset') if last_run.get('idOffset') else ''
+        last_created_time = last_run.get('lastCreatedTime') if last_run.get('lastCreatedTime') else ''
```

Comment:
```suggestion
        id_offset = last_run.get('idOffset', '')
        last_created_time = last_run.get('lastCreatedTime', '')
```
or in case the values can be `None`:
```suggestion
        id_offset = last_run.get('idOffset') or ''
        last_created_time = last_run.get('lastCreatedTime') or ''
```